integration/kind-test/configs/e2e-coredns-clusterrole.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Why are we duplicating this yaml?

Reply:
We don't have to, but I thought it would be organized if the related configs were in the specific kind-test config folder. Can delete duplicate if deeded unnecessary.

Reply:
Anything we can reuse can be moved to the shared folder.